other-builtin-functions/03_current_watermark/03_current_watermark.md

```diff
@@ -0,0 +1,78 @@
+# 03 Filtering out Late Data
+
+![Twitter Badge](https://img.shields.io/badge/Flink%20Version-1.14%2B-lightgrey)
+
+> :bulb: This example will show how to filter out late data using the `CURRENT_WATERMARK` function.
+
+The source table (`mobile_usage`) is backed by the [`faker` connector](https://flink-packages.org/packages/flink-faker), which continuously generates rows in memory based on Java Faker expressions.
+
+As explained before in the [watermarks recipe](../../aggregations-and-analytics/02_watermarks/02_watermarks.md), Flink uses watermarks to measure progress in event time. By using a `WATERMARK` attribute in a table's DDL, we signify a column as the table's event time attribute and tell Flink how out of order we expect our data to arrive.  
+
+There are many cases when rows are arriving even more out of order than anticipated, i.e. after the watermark. This data is called *late*.  An example could be when someone is using a mobile app while being offline because of lack of mobile coverage or flight mode being enabled. When Internet access is restored, previously tracked activities would then be sent.
+
+In this recipe, we'll filter out this late data using the [`CURRENT_WATERMARK`](https://ci.apache.org/projects/flink/flink-docs-release-1.14/docs/dev/table/functions/systemfunctions/) function. In the first statement, we'll use the non-late data combined with the [`TUMBLE`](../../aggregations-and-analytics/01_group_by_window/01_group_by_window_tvf.md) function to send the unique IP addresses per minute to a downstream consumer (like a BI tool). Next to this use case, we're sending the late data to a different sink. For example, you might want to use these rows to change the results of your product recommender for offline mobile app users. 
+
+This table DDL contains both an event time and a processing time definition. `ingest_time` is defined as processing time, while `log_time` is defined as event time and will contain timestamps between 45 and 10 seconds ago.   
+
+## Script
```

Comment:
nit: you could use temporary views for the late and non-late data (only applying the filter on the original source table). And then give the final sink tables more descriptive names, e.g. "unique_users_per_window" and "late_usage_events" or so.